sass/_Quickview.scss

```diff
@@ -216,6 +216,7 @@
 
 .coveo-icon-for-quickview {
   text-align: center;
+  max-width: 32px;
```

Comment:
If there is another dimension that we can relate the `32px` to, it would be good to use a sass variable.
If there isn't, this is fine.

Reply:
No didn't find any 😢